### ISSUE TITLE
iron-iconset-svg accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-iconset-svg#7]
I am unable to access the icons or alt text using a screen reader. 

I cannot navigate to the icons using the keyboard only (which is fine since they are not buttons). That said, when I use the mouse to hover over them, there is an animation that occurs, which I cannot reproduce using keyboard only. Should these be keyboard navigable then to trigger this animation?

**Comment by notwaldorf** [comment]
This can be fixed as soon as https://github.com/PolymerElements/iron-icon/issues/8 is fixed.
